maps/z2.dmm

```diff
@@ -35473,6 +35474,7 @@
 "bIv" = (
 /obj/tombstone{
 	desc = "The inscription has eroded away.";
+	icon_state = "memorials";
```

Comment:
I'd rather see these get their own tombstone subtype instead of hard coding icon states onto the maps

Reply:
Sure; I tried to look for the tombstone code to do this but couldn't find it, but I will try again.

Reply:
it's in hallooween.dm I believe

Reply:
Okay. I think I set it up correctly; I also just went through and made another headstone sprite and used that to replace the old one totally. Let me know if all that looks okay.